src/absil/il.fs

```diff
@@ -1986,7 +1986,11 @@ type ILResource =
     member r.GetBytes() = 
         match r.Location with
         | ILResourceLocation.LocalIn (file, start, len) -> 
-            FileSystem.ReadAllBytesShim(file).[start .. start + len - 1]
+            let bytes = Array.zeroCreate len
+            use fileStream = FileSystem.FileStreamReadShim(file)
+            fileStream.Position <- int64 start
+            fileStream.Read(bytes, 0, len) |> ignore
```

Comment:
Good catch!

One thing - my understanding is that this isn't guaranteed to read all the bytes, it might read less.  In any case, could you change to use ReadBinaryChunk https://github.com/Microsoft/visualfsharp/blob/master/src/absil/illib.fs#L1291.  That does the right thing